benchopt/utils/conda_env_cmd.py

```diff
@@ -22,7 +22,7 @@
   - conda-forge
   - nodefaults
 dependencies:
-  - python=3.8
+  - python=3.9
```

Comment:
```suggestion
  - python
```

Reply:
This would install the latest available python so `3.9` for now and as `3.10` roll out, it will move on naturally.